src/word_debt_bot/cogs/game_commands.py

```diff
@@ -58,11 +58,28 @@ async def register(self, ctx):
         await ctx.send("Registered with 10,000 debt!")
 
     @commands.command(name="info")
-    async def info(self, ctx):
+    async def info(
+        self,
+        ctx,
+        name: typing.Optional[str] = commands.parameter(
```

Comment:
I learned recently that you can put a [User](https://discordpy.readthedocs.io/en/stable/api.html?highlight=user#discord.User) directly as a type, and it will build the object from using mentions as arguments. I haven't been using it myself either, but might be easier than searching by name.

Reply:
ok i did it